.pipelines/templates/.builder-release-template.yaml

```diff
@@ -54,8 +54,7 @@ steps:
       SKU_NAME=${OS_VERSION} && \
       if [[ "${HYPERV_GENERATION,,}" == "v2" ]]; then SKU_NAME="${SKU_NAME}gen2"; fi && \
       if [[ ${OS_VERSION} == "V2" && ${ARCHITECTURE,,} == "arm64" ]]; then SKU_NAME="${SKU_NAME}arm64"; fi && \
-      if [[ ${OS_VERSION} == "18.04" && ${ARCHITECTURE,,} == "arm64" ]]; then SKU_NAME="${SKU_NAME}arm64"; fi && \
-      if [[ ${OS_VERSION} == "22.04" && ${ARCHITECTURE,,} == "arm64" ]]; then SKU_NAME="${SKU_NAME}arm64"; fi && \
+      if [[ (${OS_VERSION} == "18.04" || ${OS_VERSION} == "22.04" || ${OS_VERSION} == "24.04") && ${ARCHITECTURE,,} == "arm64" ]]; then SKU_NAME="${SKU_NAME}arm64"; fi && \
```

Comment:
Do we even build ARM64 for 18.04, why do we have this? Can I remove it?

Reply:
I dont think we do, should be safe to remove once you confirm with Charlie!

Reply:
If so, I can do that in a separate PR. CC: @chengliangli0918 :)

Reply:
it is safe to remove it now. We didn't build arm64 18.04 since July 2022